packages/dds/tree/src/test/forestTestSuite.ts

```diff
@@ -45,14 +45,28 @@ import {
 import { MockDependent } from "./utils";
 import { testGeneralPurposeTreeCursor } from "./cursorTestSuite";
 
+/**
+ * Configuration for the forest test suite.
+ */
+export interface ForestTestConfiguration {
+    suiteName: string;
+    factory: (schema: StoredSchemaRepository) => IEditableForest;
+
+    /**
+     * Giving errors on active cursors during edits is not required the the Forest API specification,
+     * but is nice for debugging.
+     * Performance oriented forest implementations may opt ouf of this check.
+     * Applications wanting help debugging invalid forest API use should use cursors that include this check.
+     */
+    skipCursorErrorCheck?: true;
```

Comment:
I have added a comment based on your suggestion, though I didn't use your phrasing since it implies something different (that cursors are cleared, not that uncleared cursors error, which is what is actually tested)